src/UniqueCodes.php

```diff
@@ -6,127 +6,51 @@
 
 class UniqueCodes
 {
-    /**
-     * The prime number that is used to convert a number to a unique other number within the maximum range.
-     *
-     * @var int
-     */
-    protected $obfuscatingPrime;
+    protected int $obfuscatingPrime;
```

Comment:
these are breaking changes, so we can not do those yet; I would currently keep all the docblocks and only remove those and add typehints in the next major version